### DISCUSSION TITLE
ElevenClock Hidden when Start Menu is opened but not when TranslucentTb is installed and enabled

### DISCUSSION THREAD

**Discussion by micheldiemer** [martinet101/ElevenClock#1229] [Q&A]
I have read the Wiki page, Documentation, The Actual Clock. Below the paragraph :

« This window is being constantly raised to prevent the taskbar from displaying over it. If the taskbar has the focus (for example, when the start menu is opened), the clock remains hidden, because a focused top-level window's z-index is higher than the z-index of an unfocused top-level window. This bug can't be fixed, as it is a minor issue and a fix would be difficult to implement, if not impossible. »

I have installed and enabled both ElevenClock and TranslucentTb (https://translucenttb.github.io/) and with TranslucentTb enabled, when I click on the start menu, ElevenClock is not hidden... Would it be possible to dive into it and find a solution ? 

ElevenClock is great and I love it. I display seconds, day name and week number.

**Comment by marticliment** [comment]
> I have installed and enabled both ElevenClock and TranslucentTb (https://translucenttb.github.io/) and with TranslucentTb enabled, when I click on the start menu, ElevenClock is not hidden... Would it be possible to dive into it and find a solution

If it works with TranslucentTB and not with Windows taskbar, there's some parameter that translucent tb is changing that is disabling topmost z-index. I will investigate this,
